### ISSUE TITLE
TypeError: Cannot read property 'TYPED_ARRAY_SUPPORT' of undefined

### ISSUE THREAD

**Issue by jasonLaster** [cyrus-and/chrome-remote-interface#162]
Hi, we started seeing this `TYPED_ARRAY_SUPPORT` of undefined in the [debugger-launchpad](https://github.com/devtools-html/devtools-core)

I'm going to dig into this a bit. From my initial research it looks like a dependency that was bumped or added which was not designed for the web.

#### STR

1.  clone  [debugger-launchpad](https://github.com/devtools-html/devtools-core)
2. yarn 
3. cd packages/devtools-launchpad
4. yarn start

<img width="1095" alt="screen shot 2017-06-09 at 1 04 16 pm" src="https://user-images.githubusercontent.com/254562/26986193-6fb5bda0-4d14-11e7-9ccb-31f862d9ee3d.png">

### Stack Trace
![](http://g.recordit.co/Wvq5NBXdj4.gif)

**Comment by cyrus-and** [comment]
Hm, that code seems to belong to [buffer][1]. Not sure if that's a problem with my webpack conf.

/cc @feross

[1]: https://github.com/feross/buffer/blob/master/index.js

**Comment by cyrus-and** [comment]
FYI this is the backtrace of the [`DEBUG=true` version][1] where `global` is `undefined`:

```
Uncaught TypeError: Cannot read property 'TYPED_ARRAY_SUPPORT' of undefined
    at Object.<anonymous> (chrome-remote-interface.js:879)
    at Object.module.exports.exports.byteLength (chrome-remote-interface.js:2627)
    at __webpack_require__ (chrome-remote-interface.js:21)
    at Object.<anonymous> (chrome-remote-interface.js:831)
    at __webpack_require__ (chrome-remote-interface.js:21)
    at Object.<anonymous> (chrome-remote-interface.js:52)
    at Object.module.exports.module.exports (chrome-remote-interface.js:91)
    at __webpack_require__ (chrome-remote-interface.js:21)
    at chrome-remote-interface.js:41
    at Object.<anonymous> (chrome-remote-interface.js:2)
```

[1]: https://gist.github.com/cyrus-and/9b49ccc758b81e869b7e5ef089a35a71#file-chrome-remote-interface-js

**Comment by cyrus-and** [comment]
Fixed in devtools-html/devtools-core#454.
